src/Catty/Helpers/Util/Util.swift

```diff
@@ -95,4 +95,41 @@ func synchronized(lock: AnyObject, closure: () -> Void) {
         let platformVersionWithoutPatch = "\(major).\(minor)" as String
         return platformVersionWithoutPatch
     }
+
+    class func screenSize(_ inPixel: Bool) -> CGSize {
+        var screenSize = inPixel ? UIScreen.main.nativeBounds.size : UIScreen.main.bounds.size
+
+        if inPixel && UIScreen.main.bounds.height == CGFloat(kIphone6PScreenHeight) {
```

Comment:
if you use kIphone6PScreenHeight you can delete "#define IS_IPHONEPLUS (([Util screenHeight] - kIphone6PScreenHeight) ? NO : YES)" in util.h